x-pack/plugins/reporting/server/export_types/common/get_conditional_headers.test.ts

```diff
@@ -134,25 +126,12 @@ test(`uses basePath from server if job doesn't have a basePath when creating sav
 });
 
 describe('config formatting', () => {
-  test(`lowercases server.host`, async () => {
```

Comment:
I have to look into why this test was passing in the first place. I thought the code just looks for `kibanaServer.hostname` but maybe there is/was a leak somewhere